pytket/binders/passes.cpp

```diff
@@ -1114,6 +1125,22 @@ PYBIND11_MODULE(passes, m) {
       ":param label: optional label for the pass"
       "\n:return: a pass to perform the transformation",
       py::arg("transform"), py::arg("label") = "");
+
+  m.def(
+      "CustomPassMap", &CustomPassMap,
+      "Generate a custom pass from a user-provided circuit transfomation "
+      "function."
+      "\n\n"
+      "It is the caller's responsibility to provide a valid transform."
+      "\n\n"
+      ":param transform: function taking a :py:class:`Circuit` as an argument "
+      "and returning a pair of a new transformed circuit and a pair of maps "
+      "corresponding to changes to the initial and final maps that the "
```

Comment:
```suggestion
      "corresponding to the initial and final maps that the "
```
I think it's not just changes but the whole maps?

Reply:
ah yeah, added